src/tests/Microsoft.PowerFx.Connectors.Tests/LivePublicSwaggerTests.cs

```diff
@@ -169,6 +169,33 @@ public async Task RealTest3()
             Assert.Equal("New Year's Day", ((StringValue)result).Value);
         }
 
+        [Fact(Skip = "Live Test")]
+        public async Task RealTest4()
+        {
+            var config = new PowerFxConfig();                        
+            OpenApiDocument docXkcd = await ReadSwaggerFromUrl(@"https://api.apis.guru/v2/specs/xkcd.com/1.0.0/openapi.json").ConfigureAwait(false);
+            OpenApiDocument docWorldTime = await ReadSwaggerFromUrl(@"https://api.apis.guru/v2/specs/worldtimeapi.org/20210108/openapi.json").ConfigureAwait(false);
+
+            using var clientXkcd = new HttpClient() { BaseAddress = new Uri(@"http://xkcd.com/") };
+            using var clientWorldTime = new HttpClient() { BaseAddress = new Uri(@"http://worldtimeapi.org/api/") };
+            var funcsXkcd = config.AddService("Xkcd", docXkcd, clientXkcd, new ConnectorSettings() { IgnoreUnknownExtensions = true });
+            var funcsWorldTime = config.AddService("WorldTime", docWorldTime, clientWorldTime, new ConnectorSettings() { IgnoreUnknownExtensions = true });
+
+            var engine = new RecalcEngine(config);
+
+            FormulaValue fv1 = engine.Eval(@"Xkcd.comicIdinfo0json(1)");
```

Comment:
> engine.Eval(@"Xkcd.comicIdinfo0json(1)"); [](http://example.com/codeflow?start=30&length=42)

Can we access the return result from the expression?

eg, 
`Xkcd.comicIdinfo0json(1).transcript`


`First(WorldTime.timezone()).Value`